Mianta/Feb-10-Mianta/stocks_challenge.py

```diff
@@ -0,0 +1,60 @@
+print("Challenge 3.2: Playing with the stock market")
+
+print()
+
+# Creating variables to store the current (approximate) market price of these 5 companies - Amazon, Apple, Facebook, Google and Microsoft.
+
+amazon = 3000
+apple = 100
+fb = 250
+google = 1400
+msft = 200
+
+print("Challenge 3.2.1: Taking user input")
+# asked the client their name and saved it to the variable.
+client_name = input ('Please enter youe name-')
+if client_name.isdigit() == True:
```

Comment:
Actually, client_name.isdigit() will always return false because input function will always return a string. Even if the user types in 123, it will still be stored as a string in python.

Reply:
I'll fix it.